### PR TITLE
feat(translate): templates

### PR DESCRIPTION
**Pull request by chaosmonster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ A job markdown always has to inlcude a front matter with the following informati
 |location | `Ber`/ `Fra`/ `Ham`/ `Muc` / `Pra` | Short key for the location the job will be located at | yes |
 |discipline | `Account`/ `Creative`/ `Strategy`/ `Tech`| This key is to provide the information for the discipline | yes |
 |level | `Internships`/ `Professionals`/ `Students`/ `Young Professionals` | What level of professionality is searched for | yes |
-|company | `SinnerSchrader`/ `Swipe`/ `Commerce` | TODO: do we need this? | maybe |
+|company | `SinnerSchrader`/ `Swipe`/ `Commerce`/ `Content` | TODO: do we need this? | maybe |
```
Comment (kotzendekrabbe) [inline README.md:41]: no companies needed

File: data/translations/de.yml [hunk 2]
```diff
@@ -1,2 +1,40 @@
 Ber: Berlin
 Fra: Frankfurt
+Ham: Hamburg
+Pra: Prag
+Muc: München
+Account: Beratung
+Creative: Kreation
+Strategy: Strategie
+Tech: Technik
+Internships: Praktikant*innen
```
Comment (kotzendekrabbe) [inline data/translations/de.yml:10]: please use ":" for gender neutral language

File: template/de/_detail.html [hunk 3]
```diff
@@ -1,4 +1,24 @@
-<div>
-    <h3>{{{title}}}</h3>
-    <p>{{{content}}}</p>
+<!DOCTYPE html>
```
Comment (kotzendekrabbe) [inline template/de/_detail.html:1]: german - phewphew :)

File: template/partials/contact.html [hunk 4]
```diff
@@ -1,27 +1,31 @@
...
+        <br>
         <span class="socialperson">
             <a href="mailto:jobs@sinnerschrader.com">Mail</a>
         </span>
 
         {{#if contact.xing}}
         | <span class="socialperson">
-            <a href="{{contact.xing}}">Xing</a>
-        </span> 
+            <a href="{{contact.xing}}">{{translate "contactXing" lang}}</a>
```
Comment (kotzendekrabbe) [inline template/partials/contact.html:18]: Do we really need to translate Xing? and if so, why we don't translate Mail than?

File: template/partials/en-hero.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
+<section class="mod-hero invert" data-theme="green">
```
Comment (kotzendekrabbe) [inline template/partials/en-hero.html:1]: question - we don't we use translation here and having 2 templates instead?

**Review by kotzendekrabbe — CHANGES_REQUESTED** [review]
thanks a bunch - a few questions / comments. but overall - awesome work. sooo cool <3

File: template/partials/contact.html (hunk 4 above)
Comment (chaosmonster) [reply template/partials/contact.html:18]: Good point I will add mail to it :)

File: template/partials/en-hero.html (hunk 5 above)
Comment (chaosmonster) [reply template/partials/en-hero.html:1]: (me being unsure if both language share the exact same html structure)

File: README.md (hunk 1 above)
Comment (chaosmonster) [reply README.md:41]: created an issue for this #35

File: data/translations/de.yml (hunk 2 above)
Comment (chaosmonster) [reply data/translations/de.yml]: done

File: template/de/_detail.html (hunk 3 above)
Comment (chaosmonster) [reply template/de/_detail.html:1]: :)

File: template/partials/contact.html (hunk 4 above)
Comment (chaosmonster) [reply template/partials/contact.html:18]: done

**Review by kotzendekrabbe — APPROVED** [review]

**Pull request merged**